channeldb/control_tower.go

```diff
@@ -242,3 +238,31 @@ func (p *paymentControl) Fail(paymentHash [32]byte) error {
 
 	return updateErr
 }
+
+// fetchPaymentBucket fetches or creates the sub-bucket assigned to this
+// payment hash.
+func fetchPaymentBucket(tx *bbolt.Tx, paymentHash lntypes.Hash) (
+	*bbolt.Bucket, error) {
+
+	payments, err := tx.CreateBucketIfNotExists(sentPaymentsBucket)
+	if err != nil {
+		return nil, err
+	}
+
+	return payments.CreateBucketIfNotExists(paymentHash[:])
+}
+
+// fetchPaymentStatus fetches the payment status from the bucket.  If the
+// status isn't found, it will default to "StatusGrounded".
+func fetchPaymentStatus(bucket *bbolt.Bucket) PaymentStatus {
+	// The default status for all payments that aren't recorded in
+	// database.
+	var paymentStatus = StatusGrounded
+
+	paymentStatusBytes := bucket.Get(paymentStatusKey)
+	if paymentStatusBytes != nil {
+		paymentStatus.FromBytes(paymentStatusBytes)
```

Comment:
No error return value?

Reply:
`StatusGrounded` is the state for unknown statuses.